lumicks/pylake/tests/test_correlated_stack.py

```diff
@@ -361,24 +362,65 @@ def make_description(m_red, m_blue, offsets):
     img = make_image(red_spots, spots, blue_spots, bit_depth)
     description = make_description(m_red, m_blue, offsets)
 
-    return img0, img, description            
-
-
-def test_image_reconstruction_grayscale():
-    img_args = {"spots": [[20, 30], 
-                          [50, 50], 
-                          [120, 30], 
-                          [50, 70], 
-                          [150, 60]],
-                "Tx_red": 20,
-                "Ty_red": 10,
-                "theta_red": 3,
-                "Tx_blue": 10,
-                "Ty_blue": 20,
-                "theta_blue": -3,
+    return img0, img, description        
+
+
+@pytest.fixture(scope="session")
+def tiff_dir(tmpdir_factory):
+    return tmpdir_factory.mktemp("tiffs")      
+
+
+@pytest.fixture(scope="session")
+def spot_coordinates():
+    return [[20, 30], [50, 50], [120, 30], [50, 70], [150, 60]]
+
+
+@pytest.fixture(scope="session")
+def warp_parameters():
+    return {"Tx_red": 20, "Ty_red": 10, "theta_red": 3,
+            "Tx_blue": 10, "Ty_blue": 20, "theta_blue": -3}
+
+
+@pytest.fixture(scope="session")
+def gray_image_data(spot_coordinates, warp_parameters):
+    img_args = {"spots": spot_coordinates,
+                **warp_parameters,
                 "bit_depth": 8}
+    img0, img, description = make_alignment_image_data(**img_args)
+    return img0, img, description, img_args["bit_depth"]
+
+
+@pytest.fixture(scope="session")
+def rgb_image_data(spot_coordinates, warp_parameters):
+    img_args = {"spots": spot_coordinates,
+                **warp_parameters,
+                "bit_depth": 16}
+    img0, img, description = make_alignment_image_data(**img_args)
+    return img0, img, description, img_args["bit_depth"]
+
+
+@pytest.fixture(scope="session")
+def rgb_image_bad_description(spot_coordinates, warp_parameters):
+    img_args = {"spots": spot_coordinates,
+                **warp_parameters,
+                "bit_depth": 16}
+    img0, img, description = make_alignment_image_data(**img_args)
+    description["Alignment red channel"][2] = 25
+    return img0, img, description, img_args["bit_depth"]
 
+
+@pytest.fixture(scope="session")
+def rgb_image_data_offset(spot_coordinates, warp_parameters):
```

Comment:
Maybe `rgb_alignment_image_data`? For the others too. To drive home the point that these are intended for alignment purposes?